llvm/torch-mlir
# Add e2e linalg support for aten.atan

As titled.

To test:

`python -m e2e_testing.main -v`
`cmake --build build --target check-torch-mlir-all`
